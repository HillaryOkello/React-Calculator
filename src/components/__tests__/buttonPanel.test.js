import React from 'react';
import renderer from 'react-test-renderer';
import ButtonPanel from '../buttonPanel';

it('renders correctly', () => {
  const tree = renderer
    .create(<ButtonPanel clickHandler={() => {}} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
