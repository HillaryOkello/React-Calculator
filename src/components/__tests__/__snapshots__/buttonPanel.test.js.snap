// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="panel"
>
  <div
    className="group"
  >
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      AC
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      +/-
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      %
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      /
    </button>
  </div>
  <div
    className="group"
  >
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      7
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      8
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      9
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      *
    </button>
  </div>
  <div
    className="group"
  >
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      4
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      5
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      6
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      -
    </button>
  </div>
  <div
    className="group"
  >
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      1
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      2
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      3
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      +
    </button>
  </div>
  <div
    className="group"
  >
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      0
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      .
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "orange",
          "font": "3rem 'Syne Mono'",
          "height": "100%",
          "textAlign": "center",
          "width": "25%",
        }
      }
      type="button"
    >
      =
    </button>
  </div>
</div>
`;
